test/unit/commands/fhir-test.js

```diff
@@ -61,6 +61,7 @@ test.serial.cb('The "fhir" command should list fhir resources', t => {
 
   yargs.command(list)
     .parse('list Patient --project projectId');
+  t.end();
```

Comment:
I think this will end the test before the verification completes in the callback.

Reply:
You may need to mock out there proxy module here like you did for `api.test.js`.

Reply:
Thanks - I'll re-test it with the mock.